src/tests/ffi.cpp

```diff
@@ -1753,7 +1753,10 @@ test_ffi_key_generate_ex(void **state)
     /* key protection */
     assert_rnp_success(rnp_op_generate_set_protection_cipher(keygen, "aes256"));
     assert_rnp_success(rnp_op_generate_set_protection_hash(keygen, "sha256"));
-    assert_rnp_success(rnp_op_generate_set_protection_iterations(keygen, 65536));
+    size_t iterations = 0;
+    assert_rnp_success(rnp_calculate_iterations("sha256", 500, &iterations));
+    assert_true(iterations > 65536);
+    assert_rnp_success(rnp_op_generate_set_protection_iterations(keygen, iterations));
```

Comment:
I'd rather see this in its own tiny test. The more granular we can keep the tests, the better we can parallellize test running soon.

Reply:
@dewyatt Agree. Updated and rebased.